### DISCUSSION TITLE
Consider removing the Import Error warning in import_attribute?

### DISCUSSION THREAD

**Discussion by amks1** [rq/rq#1828] [General]
Hi, 
This is in reference to: https://github.com/rq/rq/blob/42ac7d4150951b5f5fba8be11573920c1e6f6e69/rq/utils.py#L207
I use a lot of decorated functions inside classes, and my logs are filled with this warning.

According to the function doc, 
>Sometimes the failure during importing is due to a genuine coding error in the imported module
In this case, the exception is logged as a warning for ease of debugging.
The above logic will apply anyways regardless of the cause of the import error.

I'm not able to visualize a situation where the warning would be more helpful than an exception being raised if there's a genuine coding error, can someone help me out? I know I can just use a forked version, but I'd like to understand the reasoning before I do that.

**Comment by selwin** [comment]
Yeah, I think there's no reason that this should be logged. If this were to be logged at all, it should be at the `DEBUG` level. Mind opening a PR for this? @amks1
